### ISSUE TITLE
Notification - WCAG rule 2.4.9

### ISSUE THREAD

**Issue by SeamusLeonardHPE** [grommet/grommet#7613]
**Failed rule**

2.4.9
Link Purpose (Link Only)
Level AAA

A mechanism is available to allow the purpose of each link to be identified from link text alone, except where the purpose of the link would be ambiguous to users in general.

---

**Notes**

AAA
Need to give access to aria-label for link  to implementors
